src/content/cloud/namespaces.mdx

```diff
@@ -31,4 +31,16 @@ To share a namespace go to the [Okteto Cloud UI](https://cloud.okteto.com/#/?ori
 
 <p align="center"><Image src="/docs/cloud/images/share.png" alt="share a namespace" width="850" /></p>
 
-In the `Share` dialog, type the email address of the team members you want to share this namespace with. Once you press save, Okteto Cloud will update the necessary permissions and notify your teammates via email.
\ No newline at end of file
+In the `Share` dialog, type the email address of the team members you want to share this namespace with. Once you press save, Okteto Cloud will update the necessary permissions and notify your teammates via email.
+
+One of the best way to reduce cost while working is to put your namespace on sleep. A week has 168 hours and let's assume
```

Comment:
```suggestion
One of the best ways to reduce cost while working is to put your namespace in the `sleep` state. Sleeping a namespace scales your development environment down to an idle state to minimize your costs to the absolute minimum, savings dozens of hours per week in cloud costs.

You can specify a global, default threshold for automatically sleeping a namespace in the [garbage collection](/self-hosted/administration/configuration/#gc) configuration.
```

@pchico83 I assume this wording is appropriate since the costs won't be "zero". Or would they? I'm unclear on the specifics of whether it's something on our end that we wake the environment when needed and/or if the environment leaves some bare minimum processes in place for the wake operation.

In either case: @NitishKumar06 this is a non-blocking proposed change 😄 I'm proposing this language (and proposing we remove the text below) since docs are typically direct and objective and this copy comes across as more appropriate for a marketing page to promote the value of this feature (sleep/wake)

Reply:
I agree. I would reduce the scope of the PR to document the CLI commands

Reply:
Thanks! I've removed the text.